extra/CHANGES.txt

```diff
@@ -3,6 +3,7 @@
 	General improvements:
 
 	all : optimize `.bind` for instance methods (#10737)
+	lua : add lua_standalone global compiler flag (#10979)
```

Comment:
Minor nitpick: if we're sticking with `lua-standalone`, we should keep things consistent in the changelog (as with other forms of documentation) and write `lua-standalone` as the preferred form here instead of `lua_standalone`.

Reply:
From the global compiler documentation all lua flags start with `lua_` and not `lua-`. Shouldn't we therefore prefer `lua_standalone` for now from a consistency point of view ?

![image](https://user-images.githubusercontent.com/426959/226995996-c51cd40b-49e4-44dd-94c1-39c07e767efc.png)

Reply:
The manual page is generated from the `src-json/define.json` file, so this will be corrected when the manual pages are rebuilt.